packages/service-worker-mock/models/Clients.js

```diff
@@ -15,8 +15,8 @@ class Clients {
     return Promise.resolve(this.clients);
   }
 
-  openWindow(url) {
-    const client = new Client(url);
+  openWindow(url, postMessageCallback) {
```

Comment:
This does not look like part of the [spec](https://w3c.github.io/ServiceWorker/#dom-clients-openwindow).

Reply:
Same as above.